capsules/src/process_console.rs

```diff
@@ -945,6 +968,12 @@ impl<'a, A: Alarm<'a>, C: ProcessManagementCapability> uart::ReceiveClient
                 0 => debug!("ProcessConsole had read of 0 bytes"),
                 1 => {
                     self.command_buffer.map(|command| {
+                        if command[0] == 0 {
+                            //None is seen as a "user is typing" state
```

Comment:
Good practice: leave a space between the beginning of the comment "//" and the actual text comment.